app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java

```diff
@@ -0,0 +1,149 @@
+package swati4star.createpdf.fragment;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.design.widget.FloatingActionButton;
+import android.support.v4.app.Fragment;
+import android.support.v7.app.ActionBar;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.preference.PreferenceManager;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import swati4star.createpdf.R;
+import swati4star.createpdf.activity.FavouritesActivity;
+import swati4star.createpdf.customviews.MyCardView;
+
+public class FavouritesFragment extends Fragment
+        implements SharedPreferences.OnSharedPreferenceChangeListener {
+
+    private Activity mActivity;
+    private SharedPreferences mSharedPreferences;
```

Comment:
we already have this instance of SharedPreferences. Use this. Don't create new variable for this

Reply:
Yeah will remove it.

Reply:
You werent supposed to remove `mSharedPreferences`